Update Mac doc to reflect pip install from Ubuntu path

jwokaty opened the pull request:
Close #185

hpages commented:
@jwokaty 

Looks like this PR will loose some important information like the need to install the `psutil` module on the builders (needed by BBS Python code).

Also the `R CMD build destiny` test is maybe not so important but I don't think it hurts to keep it in the doc.

Finally shouldn't the doc explain what the `BBS_UBUNTU_PATH` variable is? We want to make sure that a person new to the build system will be able to use our documentation.

Thanks

jwokaty committed:
Update Mac doc to reflect pip install from Ubuntu path

jwokaty committed:
Put psutil in pps_bbs.txt, add instructions to Mac doc

jwokaty commented:
Thanks for your comments. I thought `destiny` was deprecated. I've added back in its test. I also decided to put `psutil` in a separate `Ubuntu-files/20.04/pip_bbs.txt`, which is a little overkill, but it's more organized. I've added installation instructions for it.

hpages commented:
Thank you. Looks like you're missing the triple backtick after `R CMD build destiny`.

Also, for the sake of clarity, can we simplify the terminology and stick with "Python modules"? Right now the doc uses 3 different designations for those: "Python modules", "Python packages", and "pip packages" :dizzy: 

Thanks again
